src/pipx/main.py

```diff
@@ -27,32 +28,49 @@ def print_version() -> None:
     print(__version__)
 
 
+def indented_wrap(text, subsequent_indent="", split="\n", **kwargs):
+    text = textwrap.dedent(text).strip()
+    width = shutil.get_terminal_size().columns - 2
```

Comment:
OK, one more thing just to make sure.  (I've been burned by weird `get_terminal_size` behavior before).
Just to make sure we don't have pathological formatting, can you wrap the width in a `max()` statement like:
```
    width = max(shutil.get_terminal_size().columns, 40) - 2
```

Reply:
done.